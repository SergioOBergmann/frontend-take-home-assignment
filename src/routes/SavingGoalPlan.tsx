import * as React from 'react';

import { PageTitle, InputGroup } from '../styles/global';

import house from './../icons/house.svg';

import Card from './../components/Card';

const SavingGoalPlan: React.FunctionComponent = () => {
  return (
    <>
      <PageTitle>
        {"Let's plan your "}
        <strong>saving goal.</strong>
      </PageTitle>
      <Card iconImg={house} iconText="House" title="Buy a house" />
    </>
  );
};

export default SavingGoalPlan;
