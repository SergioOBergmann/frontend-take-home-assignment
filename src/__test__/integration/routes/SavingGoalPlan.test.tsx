import * as React from 'react';

import { render, RenderResult } from '@testing-library/react';

import SavingGoalPlan from './../../../routes/SavingGoalPlan';

let documentBody: RenderResult;

describe('<SavingGoalPlan />', () => {
  beforeEach(() => {
    documentBody = render(<SavingGoalPlan />);
  });

  it('matches snapshot', () => {
    const { baseElement } = documentBody;
    expect(baseElement).toMatchSnapshot();
  });
});
