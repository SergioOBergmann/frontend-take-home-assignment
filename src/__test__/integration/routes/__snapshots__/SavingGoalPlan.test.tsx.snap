// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SavingGoalPlan /> matches snapshot 1`] = `
<body>
  <div>
    <span
      class="sc-htpNat daZEoS"
    >
      Let's plan your 
      <strong>
        saving goal.
      </strong>
    </span>
    <div
      class="sc-gzVnrw fGSdvJ"
    >
      <img
        alt="House"
        class="sc-htoDjs jMUYuv"
        src="test"
      />
      <span
        class="sc-dnqmqq fvEnnj"
      >
        Buy a house
      </span>
      <span
        class="sc-iwsKbI hNxSrM"
      >
        Saving Goal
      </span>
      <button
        class="sc-bZQynM sc-gZMcBi dNYqZl"
      >
        Confirm
      </button>
    </div>
  </div>
</body>
`;
